zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/metric/ZkClientMonitor.java

```diff
@@ -38,9 +38,13 @@
 import org.apache.helix.monitoring.mbeans.dynamicMBeans.SimpleDynamicMetric;
 import org.apache.helix.monitoring.mbeans.exception.MetricException;
 import org.apache.helix.zookeeper.zkclient.ZkEventThread;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 
 public class ZkClientMonitor extends DynamicMBeanProvider {
+  private static final Logger LOG = LoggerFactory.getLogger(ZkClientMonitor.class);
```

Comment:
_logger exists in the super class. There is no need to add a new one here.

Reply:
fixed.